src/lib/db/feature-toggle-store.js

```diff
@@ -260,6 +260,37 @@ class FeatureToggleStore {
         return tag;
     }
 
+    async getAllFeatureTags() {
+        const rows = await this.db(FEATURE_TAG_TABLE).select(
+            FEATURE_TAG_COLUMNS,
+        );
+        return rows.map(row => {
+            return {
+                featureName: row.feature_name,
+                tagType: row.tag_type,
+                tagValue: row.tag_value,
```

Comment:
not relevant for this PR, but it really bugs me that we end up doing snake_case to camelCase conversions all the time.

Reply:
Maybe https://www.npmjs.com/package/camelcase is an option?

Reply:
https://github.com/sindresorhus/camelcase-keys

Reply:
I'm wondering if having proper types would help, could state that all stores implement a column name <~> jsonName translator so snake_case is internal to store while we use camelCase for arguments to- and return objects from store methods?

Reply:
Though this PR is probably not the place to fix it. :)